### ISSUE TITLE
No `src/rct/CMakeLists.txt`

### ISSUE THREAD

**Issue by egarrulo** [Andersbakken/rtags#371]
Hello.  I have followed the *TLDR Quickstart* for *RTags 2.0* without success.  Here is a log:

    $ pwd
    ~/Downloads/rtags-2.0
    $ cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_EXPORT_COMPILE_COMMANDS=1
    -- The C compiler identification is GNU 4.9.2
    -- The CXX compiler identification is GNU 4.9.2
    -- Check for working C compiler: /usr/bin/cc
    -- Check for working C compiler: /usr/bin/cc -- works
    -- Detecting C compiler ABI info
    -- Detecting C compiler ABI info - done
    -- Detecting C compile features
    -- Detecting C compile features - done
    -- Check for working CXX compiler: /usr/bin/c++
    -- Check for working CXX compiler: /usr/bin/c++ -- works
    -- Detecting CXX compiler ABI info
    -- Detecting CXX compiler ABI info - done
    -- Detecting CXX compile features
    -- Detecting CXX compile features - done
    -- LLVM llvm-config found at: /usr/bin/llvm-config-3.5
    -- Using Clang version 3.5.0 from /usr/lib/llvm-3.5/include/clang-c/
    -- Found Curses: /usr/lib/x86_64-linux-gnu/libcurses.so  
    CMake Error at src/CMakeLists.txt:52 (add_subdirectory):
      The source directory

        ~/Downloads/rtags-2.0/src/rct

      does not contain a CMakeLists.txt file.

    CMake Error at src/CMakeLists.txt:54 (include):
      include could not find load file:

        ~/Downloads/rtags-2.0/src/rct/compiler.cmake

    -- Configuring incomplete, errors occurred!
    See also "~/Downloads/rtags-2.0/CMakeFiles/CMakeOutput.log".

The CMakeOutput.log is big, therefore I will attach it only if it is needed.

Thanks for your attention

**Comment by cslux** [comment]
Hey @egarrulo, this happens if you download rtags(-2.0) as tarball. You have to download [rct](https://github.com/Andersbakken/rct/tree/76ffaded094b188360c12bef60f5cac9491e6c2e) too and than you have to put the content of the rct-master folder into rtags-2.0/src/rct.

However, I wouldn't recommend it, as rct should be in sync to a specific commit to rtags, otherwise strange side affects can happen (crashes, build failure etc.)

Try this instead.
```
cd ~/Downloads
git clone https://github.com/Andersbakken/rtags.git
cd rtags 
git submodule init
git submodule update
mkdir build
cd build 
cmake ../ -DCMAKE_BUILD_TYPE=Release -DCMAKE_EXPORT_COMPILE_COMMANDS=1
make
```

Regards,
Christian

**Comment by Andersbakken** [comment]
Christian is correct. Using git is the preferred option at this point.
We should look into providing proper tarballs. I'll see if I can figure
something out.

Anders

**Comment by egarrulo** [comment]
Thanks.  Now it works.  I have met these differences versus the current instructions:

-  `rdm` and `rc` are in the *bin* folder, therefore -- after `make` -- I typed `bin/rdm &` and `bin/rc -J .` instead;

- I added the *bin* folder to the `exec-path` in Emacs;

- I used ``rtags-find-symbol-at-point` because `rtags-follow-symbol-at-point` is not defined anymore.

> Christian is correct. Using git is the preferred option at this point. We should look into providing proper tarballs.

Then the README could simply say that users must be use Git to download RTags.  If I had read that, then I wouldn't have tried with the tarball.
